### PR TITLE
horizontal position is added, vertical is redone with another parent

### PR DESCRIPTION
**Pull request by Erball-M**
Снимок экрана 2024-04-12 145841
Снимок экрана 2024-04-12 145920

fix for #30

### CODE REVIEW

File: src/lib/components/DropDown.svelte [hunk 1]
```diff
@@ -71,9 +80,6 @@
 </div>
 
 <style>
-	.container {
-		position: relative;
-	}
```
Comment (RB-Lab) [inline src/lib/components/DropDown.svelte:76]: Nah, this is a bad idea. Try scrolling with open dropdown.

File: src/lib/components/DropDown.svelte [hunk 2]
```diff
@@ -39,13 +39,22 @@
...
+			let top = containerRect.bottom - parentRect.top + gap
+			let left = containerRect.left + containerRect.width / 2 - dropdownRect.width / 2 - gap
+			if (left + dropdownRect.width > parentRect.right) {
+				left = containerRect.left - dropdownRect.width
 			}
+			if (top + dropdownRect.height > viewHeight) {
+				top = containerRect.top - parentRect.top - dropdownRect.height - gap
+			}
+			dropdownDiv.style.top = `${top}px`
+			dropdownDiv.style.left = `${left}px`
```
Comment (RB-Lab) [inline src/lib/components/DropDown.svelte:57]: Okay, at this point, I guess it's time to use [popper.js](https://popper.js.org/docs/v2/)

**Review by RB-Lab — CHANGES_REQUESTED** [review]
Good start, but I don't like the positions. Should be like this:
image

**Comment by Erball-M** [comment]
this branch was merged to EM/taskDescription ( PR #24 )

**Review by RB-Lab — APPROVED** [review]

**Pull request merged**